docs/types/reference-types.rst

```diff
@@ -106,13 +106,24 @@ Array elements can be of any type, including mapping or struct. The general
 restrictions for types apply, in that mappings can only be stored in the
 ``storage`` data location and publicly-visible functions need parameters that are :ref:`ABI types <ABI>`.
 
+It is possible to mark arrays ``public`` and have Solidity create a :ref:`getter <visibility-and-getters>`.
```

Comment:
I don't think this is a good place to put it. This section is about arrays as concepts of types, not about state variables. Put here, it reads like you could add `public` to any array type.

Reply:
Ah, I see this was only moved. So in this case, we should clarify that it only works on state variables.

Reply:
@chriseth Previously this was at the bottom of the section, also in a place that didn't make much sense. I have amended, but there are other examples of text very similar to this in other places in the docs, where it's equally unclear, so I might do a clarification around all of those.